### PR TITLE
Sophos heartbeat

### PR DESCRIPTION
**Pull request by rajanjoshigl**

### CODE REVIEW

File: lrlog/tests/lrlog_test.go [hunk 1]
```diff
@@ -0,0 +1,128 @@
+package tests
```
Comment (Himanshu8859) [inline lrlog/tests/lrlog_test.go]: which file is this?

File: lrlog/lrlog.go [hunk 2]
```diff
@@ -0,0 +1,143 @@
+package lrlog
```
Comment (Himanshu8859) [inline lrlog/lrlog.go]: which file?

**Review by Himanshu8859 — CHANGES_REQUESTED** [review]

File: lrlog/tests/lrlog_test.go (hunk 1 above)
Comment (rajanjoshigl) [reply lrlog/tests/lrlog_test.go]: This logging package is used by heartbeat package

File: lrlog/lrlog.go (hunk 2 above)
Comment (rajanjoshigl) [reply lrlog/lrlog.go]: This logging package is used by heartbeat package

File: heartbeat/status_beater.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+			Seconds: now / time.Nanosecond.Nanoseconds(),
+		},
+		Status: Status{
+			Code:        status,
+			Description: description,
+		},
+	}
+	msgJSON, err := json.Marshal(msg)
+	if err != nil {
+		lrlog.Warning(errors.Wrap(err, "internal heartbeat message json conversion failed"))
```
Comment (Himanshu8859) [inline heartbeat/status_beater.go]: you can use logp library for warning like logp.Warn()

**Review by Himanshu8859 — CHANGES_REQUESTED** [review]

**Pull request merged**